Contribute/index.md

```diff
@@ -0,0 +1,140 @@
+- `D-Team` - this label is usually used when the PR is about something with Umbraco Core that the documentation team is not familiar with, meaning it requires a review from the Umbraco development team
+- `Umbraco Cloud`, `Umbraco Forms`, etc - if the issue is specific to something other than Umbraco CMS it will get the relevant Umbraco product label
+
+## Markdown conventions
+The Umbraco Documentation uses Markdown for all of the documentation - but more precisely we use the CommonMark specification, read more about the difference [here](https://commonmark.org/).
+
+### Structure
+For the documentation project, each individual topic is contained in its own folder.
+Each folder must have an `index.md` file which links to the individual sub-pages, if images are used, these must be in `images` folders next to the .md file referencing them relatively.
+
+* topic
+	* images
+		* images.jpg
+	* Subtopic
+		* images
+		* index.md
+	* index.md
+	* otherpage.md
+
+### Images
+Images are stored and linked relatively to .md pages, and should by convention always be in an `images` folder. So to add an image to `/documentation/reference/partials/renderviewpage.md` you link it like so:
+
+	![My Image Alt Text](images/img.jpg)
+
+And store the image as `/documentation/reference/partials/images/img.jpg`
+
+Images can have a maximum width of **800px**. Please always try to use the most efficient compression, `gif`, `png` or `jpg`. No `bmp`, `tiff` or `swf` (Flash).
+
+### External links
+Include either the complete URL, or link using Markdown:
+
+	https://yahoo.com/something
+
+	or
+
+	[yahoo something](https://yahoo.com/something)
+
+
+### Internal links
+If you need to link between pages, always link relatively, and include the .md extension.
+
+	[Umbraco.Helpers](Umbraco.Helpers.md)
+
+	or
+
+	[Umbraco.Helpers](../../Reference/Umbraco.Helpers.md)
+
+### Formatting code
+Indent your sample with a single tab, which will cause it to be rendered as `<pre><code>` tags.
+For inline code, wrap in ` (backtick) chars.
+
+Use # for the headline, ## for sub headers and ### for parameters (on code reference pages)
+
+For optional parameters wrap in _ (underscore) - end result: `###_optionalParameter_`
+
+ ## Contribution badge
+If you make a pull request to any Umbraco repository that gets merged in you will get a Contributor badge on your member profile on Our:
+
+![Contributor badge on our](images/c-trib-badge.png)
+
+The Documentation Curators will search for your profile when merging a PR in and add the badge, but if it is forgotten just make a comment on the closed PR and we will do it as soon as possible!
+
+ ## Documentation Curators
+ All the work of adding labels, going through issues and PR's and managing the Documentation repository is done by the Umbraco Documentation Curators team. If you wish to know more about who they are and how they work there is some information about them here: https://our.umbraco.com/community/the-documentation-curators/
```

Comment:
Ask @Jeavon if we should add more information gathered during our first meetings on how we work.  I think it is worth putting a small resume here on when we meet and what we do exactly.

Reply:
... and @sofietoft and @marcemarc  :-)